chain/proofs/ffi/verifier_cgo.go

```diff
@@ -0,0 +1,55 @@
+//go:build cgo
```

Comment:
this is a minimised version of ./storage/sealer/ffiwrapper/verifier_cgo.go that has just what we need